packages/shared/src/models/IRecipeModelIndex.ts

```diff
@@ -0,0 +1,4 @@
+export interface RecipeModelIndex {
```

Comment:
What is the purpose of this interface ?

Reply:
Represents the type of the key of the `ApplicationRegistry` class:

```
export class ApplicationRegistry<T extends RecipeModelIndex> {
```